src/js/utils.js

```diff
@@ -47,15 +47,50 @@ const changeAnchorLink = (anchorElement, { user, repository, typeOfSearch, date
   }
 }
 
-export const validateForm = (formData, anchorElement) => {
-  const filterData = createSearchFilter(formData);
+export const validateForm = () => {
+  const fielValid = {
+    type_of_search: {
+      required: true,
+    },
+    input_user: {
+      required: true,
+    },
+    input_repository: {
+      required: true,
+    },
+    input_condition: {
+      required: true,
+    },
+    input_date: {
+      required: true,
+    },
+  };
+  const fiels = document.querySelectorAll(
+    ".box.d-inline input,.box.d-inline select"
+  );
```

Comment:
I think you can do it in one line, what do you think?

```suggestion
  const fiels = document.querySelectorAll(".box.d-inline input,.box.d-inline select");
```

Reply:
Also remember that when using JS it's recommended to use "id" on the html, so also the selector is easier